doc/user-guide/user-guide.rst

```diff
@@ -76,38 +64,117 @@ At the same time, all images are annotated with the version information about th
 .. _Core-packages:
 
 Core packages
--------------
+=============
+
+The following is a list of the core packages that are used to define a particular cycle.
 
+  - ts_ddsconfig: Contains DDS Quality of Service (QoS) configuration and various sanctioned OpenSplice configuration options for running systems in shared memory, single process and unit testing.
   - ts_xml: Contains the definition of the interfaces of all components in the system.
   - ts_sal: Contains the core DDS communication middleware layer.
   - ts_idl: Contains ancillary files used by ts_salobj and other higher level applications.
   - ts_salobj: High-level Python library to develop CSCs and SAL Script.
 
 
-.. _Building-a-new-Cycle-or-Revision:
+.. _Building-a-New-Cycle-or-Revision:
 
-Building a new Cycle or Revision
+Creating a New Cycle or Revision
 ================================
 
-There are two ways users can run a build; manually on a private server or using the `TSSW Jenkins server`_.
-Since the build requires credentials for the nexus servers, both to access the licensed version of OpenSplice and to pull/push images to the nexus docker registry, it is advisable to use the `TSSW Jenkins server`_.
+The following is the procedure used to create and build a new cycle or revision.
+
+.. _Building-a-new-Cycle-or-Revision-Workflow:
+
+Workflow
+--------
+
+In general, it is expected that new cycle builds shall be deployed in one of the integration test stands to be tested before rolling to production (e.g. summit).
+Because the validation process can take some time and since summit environment is in constant need for updates (new revisions), a new cycle can only be considered officially released after the production rollout.
+
+To deal with this process in a more comprehensive way, this repository uses a different workflow than the  one adopted by `TSSW`_.
```

Comment:
remove extra space between 'the' and 'one'. Only effects the text file and not the rendered page